scripts/release/atomic_pipeline.py

```diff
@@ -71,8 +72,9 @@ def build_image(
         logger.info("Logging in MongoDB Artifactory for Garasign image")
         mongodb_artifactory_login()
         logger.info("Signing image")
-        sign_image(build_configuration.registry, build_configuration.version)
-        verify_signature(build_configuration.registry, build_configuration.version)
+        for registry in build_configuration.registry:
```

Comment:
Should this be build_configuration.registries?

Reply:
or in other words, I think we should rename `build_configuration.registry` to `build_configuration.registries`.